internal/sdkprovider/service/kafka/mirrormaker_replication_flow_test.go

```diff
@@ -275,6 +278,10 @@ func testAccCheckAivenMirrorMakerReplicationFlowAttributes(n string) resource.Te
 			return fmt.Errorf("expected to get a correct enable from Aiven")
 		}
 
+		if a["exactly_once_delivery_enabled"] != "true" {
+			return fmt.Errorf("expected to get a correct exactly_once_delivery_enabled from Aiven")
+		}
+
```

Comment:
nit: no need to add the check here as we already check the attribute in the test body